src/App.tsx

```diff
@@ -1,16 +1,57 @@
 import React from 'react';
 import './App.scss';
 
+type AppState = {
+  count: number;
+};
+
 class App extends React.Component {
-  addOne = () => {};
+  state: AppState = {
+    count: 0,
+  };
+
+  addOne = () => {
+    this.setState((state: AppState): AppState => ({
+      count: state.count + 1,
+    }));
+  };
 
-  add100 = () => {};
+  add100 = () => {
+    this.setState((state: AppState): AppState => ({
```

Comment:
тут можна не вказуваnи тип для state якщо ти його опишиш для класу
```suggestion
    this.setState(currentState => ({
```